### ISSUE TITLE
No mail confirmation possible for certain providers

### ISSUE THREAD

**Issue by sblauth** [readthedocs/readthedocs.org#7501]
## Details

I recently created an account for readthedocs, but I am unable to get certain mail adresses, ending in either @gmx.de or @web.de verified. I tried re-sending the confirmation several times, also removing and adding the addreses back again does not change this behavior. The mail was also not placed in a spam folder or something similar. However, it worked fine with another provider, using my @gmail.com adress.

* Read the Docs project URL: cashocs.readthedocs.io/
* Build URL (if applicable):
* Read the Docs username (if applicable): plugged

## Expected Result

Confirmation links are sent out to @gmx.de and @web.de mail adresses.

## Actual Result

There are no confirmation links sent to these adresses.

**Comment by humitos** [comment]
I'm not sure what we can do here from the Read the Docs side. The emails are definitely going out but maybe they are blocked by those providers for some reason? You may want to contact them and in case they tell you what's the potential reason, we could maybe change some config in our side.

**Comment by sblauth** [comment]
Thanks for your fast reply. I will contact the mail providers for additional information, and come back to you once I know more about the issue.

**Comment by ptrpfn** [comment]
+1 Same here, but with email and domain hosted by domainfactory.

**Comment by iceback** [comment]
This I got from local Mail IT group:
TL/DR: the group sending the message on behalf of RTD has gained a bad reputation (spammer, or worse)

I am seeing about 15 emails everyday from this IP Address, up until the end of day on the 10th.  After that there is nothing.  I changed the search from look for messages to looking for rejected connection.  Starting on the 11th, and ever since then, I see lots of rejected connections showing:
" REJECT sender group BLACKLIST match sbrs[-10.0:-1.5] SBRS -3.0 sender IP 149.72.39.137"
This indicates that on a reputation scale of -10 to 10, we block anything that is -1.5 and below.  It appears ReadTheDocs uses a service called sendgrid.net to send out these emails.   sendgrid.net somehow got their reputation dinged enough on the 10th to drop it below the -1.5 threshold.
Until the clean up their reputation we will not be able to get any mail from them.
This setting is controlled by the Information Security Office and was settled on after decades of following industry standards and then tweaking that depending on if we were getting too much garbage or getting too much valid stuff blocked.  We have finally found a "sweet spot" that has served us well for the last few years.  I don't think ISO will change that threshold at this point.  However, if you want, we can petition them to whitelist this IP Address if we can vouch for them as a valid sender.  But really the responsibility falls on them to maintain a good reputation score.

**Comment by stsewd** [comment]
This is still valid, I think we were trying to reach to sendgrid.

**Comment by stsewd** [comment]
Come on bot!

**Comment by humitos** [comment]
We are handling this internally with SendGrid support but there is no much we can do here. I don't think it worth to keep this issue open since there is nothing actionable from our side.

**Comment by iceback** [comment]
We, the affected, would like to know when this issue with SendGrid is resolved as we may need to inform our respective IT (mail) departments. And also we will need to initiate a new email confirmation.
This thread seem the appropriate place for this

**Comment by stsewd** [comment]
We just received an email from sengrid

> Thank you very much for your patience while our compliance team has been in the works to resolve your IP issue with Spamhaus.

Can you check the emails are arriving now? (not sure if your IT dep need to update the list or something)

**Comment by stsewd** [comment]
Sure

**Comment by sblauth** [comment]
I just tried it with the @gmx.de mail address and one from work, which also did not work previously, and I can confirm that the mails  are coming now. 

Thanks a lot!

**Comment by iceback** [comment]
On seeing the success of others, I tried mine and it too works.  Case 
closed, imo.
